### PR TITLE
chore: add inclusive terms plugin

### PR DESCRIPTION
**Pull request by Abhilasha3112**
**Describe your changes**

- This pr adds eslint plugin for reporting non-inclusive terms in the codebase and suggests alternatives.

- Uses https://github.com/muenzpraeger/eslint-plugin-inclusive-language

- Also, adds stylelint and eslint check within `npm test`

**Testing performed**
Tested by running `npm test` several times.

**Additional context**
Developers can add more non-inclusive terms to the list in `.eslintrc.yml` file.

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -45,6 +46,15 @@ rules:
   react/no-unescaped-entities: 0
   import/no-unresolved: error #re-add no-unresolved
   unicorn/custom-error-definition: error
+  inclusive-language/use-inclusive-words:
+    - error
```
Comment (portenez) [inline .eslintrc.yml:50]: Can we start with a populated list? There are terms that we definitely should reject from the beginning. I can think of:

- master
- slave
- dummy

We probably should have a list.

File: package-lock.json [hunk 2]
```diff
@@ -1,16972 +1,8 @@
 {
   "name": "coffeechat",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (portenez) [inline package-lock.json]: Can you re-generate with `npm v7` ? either using nvm or a docker container?

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "An app for helping mentors and mentees connect and schedule time to meet for coffee or any comparable beverage.",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "test": "npm run eslint && npm run stylelint && jest",
```
Comment (portenez) [inline package.json]: [optional] we probably can leave all of these as separate scripts. We'll set up CI using GitHub actions, and we can have different checks per tool. 
We could also leave them here until we actually setup Github actions.

**Review by portenez — CHANGES_REQUESTED** [review]
Some actions to take:

- Regenerate the lock file with npm v7 (lockfile to v2)
- Populate the list of terms
- and an optional in there (up2u)

File: .eslintrc.yml (hunk 1 above)
Comment (Abhilasha3112) [reply .eslintrc.yml:50]: the above terms are already included by default in the plugin https://github.com/muenzpraeger/eslint-plugin-inclusive-language/blob/primary/lib/config/inclusive-words.json . Hence we do not have to include them separately.

**Review by portenez — DISMISSED** [review]

**Pull request closed without merging**